spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/AbstractCassandraJoin.scala

```diff
@@ -0,0 +1,190 @@
+package com.datastax.spark.connector.rdd
+
+import org.apache.spark.metrics.InputMetricsUpdater
+
+import com.datastax.spark.connector._
+import com.datastax.spark.connector.writer._
+import com.datastax.spark.connector.rdd.reader._
+import com.datastax.spark.connector.util.CqlWhereParser.{ EqPredicate, InListPredicate, InPredicate, RangePredicate }
+import com.datastax.spark.connector.util.{ CountingIterator, CqlWhereParser }
```

Comment:
Spaces here between imports and brackets do not seem consistent with the connector code base. I say we should try to be as consistent as possible. What do you think?